erpnext/setup/setup_wizard/operations/install_fixtures.py

```diff
@@ -341,13 +341,14 @@ def add_uom_data():
 				"category_name": _(d.get("category"))
 			}).insert(ignore_permissions=True)
 
-		uom_conversion = frappe.get_doc({
-			"doctype": "UOM Conversion Factor",
-			"category": _(d.get("category")),
-			"from_uom": _(d.get("from_uom")),
-			"to_uom": _(d.get("to_uom")),
-			"value": d.get("value")
-		}).insert(ignore_permissions=True)
+		if not frappe.db.exists("UOM Conversion Factor", {"from_uom": _(d.get("from_uom")), "to_uom": _(d.get("to_uom"))}):
```

Comment:
If you're doing this check, then you can just rerun the old patch. It'll ignore everything else anyway.

Reply:
old patch delete data from UOM Conversion factor
if not frappe.db.a_row_exists("UOM Conversion Factor"):
so user created UOM will get deleted

Reply:
Ah, alright